strings/names/last.txt

```diff
@@ -1,851 +1,1048 @@
-	Ramos
+	Jesse
 	Jewell
-	Endsley
-	Briner
-	Jenner
-	Saylor
-	Bash
-	Blyant
-	Rhinehart
-	Prescott
-	Kirk
-	Picard
-	Riker
-	Spock
-	Skywalker
-	Vader
-	Muggins
-	Buttersworth
-	Stamos
-	Sagan
-	Hawking
-	Dawkins
-	Pain
-	Bombsky
-	Fizzlebeef
-	Blowfist
-	Punchbeef
-	Buttsteak
-	Sideiron
-	Plankchest
-	Killaslot
-	Slaughter
-	Power
-	Hardpeck
-	Smith
-	Jones
-	Williams
-	Brown
-	Taylor
-	Davies
-	Wilson
-	Evans
-	Thomas
-	Roberts
+	Ji
+	Jia
+	Jiang
+	Jin
+	Joghs
 	Johnson
-	Walker
-	Wright
-	Robinson
-	Thompson
-	Hughes
-	White
-	Edwards
-	Hall
-	Patel
-	Green
-	Martin
-	Lewis
-	Wood
-	Jackson
-	Clarke
-	Harris
-	Clark
-	Scott
-	Turner
-	Hill
-	Moore
-	Cooper
-	Morris
-	Ward
-	Watson
-	Morgan
-	Anderson
-	Harrison
-	King
-	Campbell
-	Young
-	Mitchell
-	Baker
-	James
+	Jojo
+	Jones
+	Jorgensen
+	Jowers
+	Joyce
+	Judge
+	Jyllian
+	Kaʻaukai
+	Kadel
+	Kalb
+	Kanaan
+	Kanaga
+	Kaname
+	Kassab
+	Kassis
+	Kato
+	Kattan
+	Katz
+	Kaur
+	Kazem
+	Keener
+	Kelley
+	Kellogg
 	Kelly
-	Allen
-	Bell
-	Phillips
+	Kemble
+	Kemerer
+	Kenyatta
+	Keppel
+	Kepplinger
+	Key
+	Khan
+	Khosa
+	Khouri
+	Khoury
+	Kiefer
+	Kifer
+	Killaslot
+	Kim
+	Kimple
+	Kimura
+	King
+	Kirk
+	Klein
+	Knapenberger
+	Knapp
+	Kobayashi
+	Koepple
+	Kojima
+	Kondo
+	Koster
+	Kouri
+	Koury
+	Kovacevic
+	Kowalski
+	Kumar
+	Kuster
+	Kuznetsov
+	Laborde
+	Lacon
+	Lafortune
+	Laine
+	Lambert
+	Lang
+	Langston
+	Larson
+	Lauffer
+	Laurenzi
+	Lavender
+	Leach
 	Lee
-	Stewart
-	Miller
-	Parker
-	Simpson
-	Bennett
-	Davis
-	Griffiths
-	Shaw
-	Price
-	Cook
-	Richardson
-	Murray
+	Leech
+	Leichter
+	Leslie
+	Lester
+	Lettuce
+	Levett
+	Lewis
+	Li
+	Liang
+	Light
+	Liao
+	Lin
+	Lindberg
+	Lineman
+	Linton
+	Llora
+	Lloyd
+	Logue
+	Lombardi
+	Lopez
+	Lord
+	Losey
+	Lowe
+	Lowstetter
+	Lucy
+	Ludwig
+	Lugassi
+	Luna
+	Luo
+	Lynch
+	Lysenko
+	Maalouf
+	Maclagan
+	Magnusson
+	Magor
+	Mahajan
+	Mahal
+	Maharaj
+	Majumdar
+	Malhotra
+	Maloof
+	Malouf
+	Manos
+	Marcotte
+	Margaret
+	Maroun
+	Marriman
 	Marshall
-	Begum
-	Murphy
-	Khan
-	Gray
-	Collins
-	Bailey
-	Carter
-	Robertson
-	Graham
-	Adams
-	Richards
-	Cox
-	Singh
-	Hussain
-	Ellis
-	Wilkinson
-	Foster
-	Thomson
-	Russell
-	Ali
-	Reid
+	Martin
+	Martinez
+	Mary
+	Masih
 	Mason
-	Chapman
-	Powell
-	Owen
-	Ahmed
-	Gibson
-	Rogers
-	Webb
-	Holmes
-	Mills
+	Mathews
+	Mathis
+	Matsuda
+	Matsumoto
 	Matthews
-	Hunt
-	Palmer
-	Lloyd
-	Kaur
-	Fisher
-	Ivanov
-	Smirnov
-	Vasilyev
-	Petrov
-	Kuznetsov
-	Mikhaylov
-	Pavlov
-	Semenov
-	Andreev
-	Alekseev
-	Turnbuckle
-	Hampus
-	the Third
-	the Fourth
-	the Eighth
-	the Twentieth
-	Garcia
-	Fernandez
-	Gonzalez
-	Rodriguez
-	Lopez
-	Gomez
-	Driscoll
-	Foley
+	Mayberry
 	McNab
-	O'Malley
-	Lynch
-	Moloney
-	Bologna
-	Scully
-	Wagner
-	Bauer
-	Meier
-	Schneider
-	Jaeger
-	Metzger
-	Gaumond
-	Walmund
-	Hubert
-	Hanlon
-	Agnor
-	Dardar
-	Alvin
-	Samsa
-	Gruber
-	Novik
-	Jafarov
-	Dubois
-	Dupont
-	Lambert
+	Mccloud
+	Mcclymonds
+	Mccullough
+	Mccune
+	Mcdonald
+	Mcfall
+	Mcintosh
+	Mckendrick
+	Mcloskey
+	Mcmullen
```

Comment:
if you're going to have McNames, spell them right
```suggestion
	McCloud
	McClymonds
	McCullough
	McCune
	McDonald
	McFall
	McIntosh
	McKendrick
	McCloskey
	McMullen
```

Reply:
Those were present in the original name file, I just ordered them. But I can fix them to be proper